FindDict/FindDict/Sources/Scences/Game/GameVC.swift

```diff
@@ -50,9 +50,18 @@ final class GameVC: ViewController {
             }
         }
     }
-    
     private let cropImageView: UIImageView = UIImageView()
     
+    private var cropImage: UIImage = UIImage(){
```

Comment:
저도 인지하지 못하고 있었는데 crop이 동사이다보니 `동사 + 목적어` 어순이 되어 변수보다 함수명으로 이해하게 되는 것 같습니다! 
```suggestion
    private var cropImage: UIImage = UIImage(){
```
croppedImage로 변수명이 `수식부 + 명사` 형태가 되게 수정해주시면 더 좋을 것 같습니다!

Reply:
넵!!!